curvefs/src/client/s3/client_s3_cache_manager.cpp

```diff
@@ -392,9 +412,66 @@ int FileCacheManager::HandleReadRequest(
         dataCache.first->AddReadDataCache(dataCache.second);
         VLOG(9) << "add read data cache end";
     }
+
     return 0;
 }
 
+void FileCacheManager::PrefetchS3Objs(std::vector<std::string> prefetchObjs) {
+    uint64_t blockSize = s3ClientAdaptor_->GetBlockSize();
+    uint32_t prefetchBlocks = s3ClientAdaptor_->GetPrefetchBlocks();
+    for (auto &obj : prefetchObjs) {
+        std::string name = obj;
+        // 加锁
```

Comment:
in english.And this comment is not necessary, unless you describe it in more detail